portlets/src/main/frontend/src/apps/components/badge/BadgeList.vue

```diff
@@ -140,7 +140,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                       v-model="editedbadge.enabled"
                       :disabled="editedbadge.domainDTO==null||!editedbadge.domainDTO.enabled"
                       type="checkbox">
-                    <span class="slider round"></span>
+                    <span class="slider round"><span class="absolute-yes">{{ $t(`exoplatform.gamification.YES`,"YES") }}</span></span>
```

Comment:
Is there a need to add an extra SPAN element here ?

Reply:
Yes, we must add it because we need to apply specific css to this span.

Reply:
Couldn't we merge both SPAN elements?

Reply:
We cannot merge two spans because each span has a specific css